### ISSUE TITLE
Updating min ruby version outside of a major release is disruptive

### ISSUE THREAD

**Issue by jmoody** [supermarin/xcpretty#186]
This commit 91c766f4c82541564297e958254ba5d920166cf5 bumped the min ruby version to 2.0.

In my opinion, you should have updated the _major_ version for such a change - this change causes all kinds of headaches for me.

I realize you are still in the 0.x stage, but I think this gem is ready for 1.x.

Keep up the great work!  This gem is great.

**Comment by ikesyo** [comment]
I think it's OK because: the change is introduced in v0.2.0, and in terms of Semantic Versioning, pre-1.0 software can break backwards compatibility in its minor version updates.

**Comment by jmoody** [comment]
I agree that making this change in a 0.2.0 was a good idea and it does not violate semver. 0.2.0 sends the right signal to users - something significant has changed.

However, semver also says:

> If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you're worrying a lot about backwards compatibility, you should probably already be 1.0.0.

I expect I am not the only one who uses this gem in production.

I am not blameless here.  I maintain a several gems that cannot update to 1.0 for marketing reasons and it causes me all kinds of headaches.  I guess I am encouraging you to update to a 1.0 version - it will give you and your users more flexibility.
